internal/dao/rs.go

```diff
@@ -127,3 +127,50 @@ func (r *ReplicaSet) Rollback(fqn string) error {
 
 	return nil
 }
+
+// GetOwners returns the owners of the resource.
+func (r *ReplicaSet) GetOwners(path string) ([]OwnerInfo, error) {
+	var owners []OwnerInfo
+
+	rs, err := r.GetInstance(path)
+	if err != nil {
+		return nil, err
+	}
+	ownerRefs := rs.GetObjectMeta().GetOwnerReferences()
+
+	for _, ownerRef := range ownerRefs {
+		gvr, namespaced, err := GVRForKind(ownerRef.APIVersion, ownerRef.Kind)
+		if err != nil {
+			return nil, err
+		}
+
+		var ownerFQN string
+		if namespaced {
+			ownerFQN = FQN(rs.Namespace, ownerRef.Name)
+		} else {
+			ownerFQN = ownerRef.Name
+		}
+
+		owners = append(owners, OwnerInfo{
+			GVR: gvr,
+			FQN: ownerFQN,
+		})
+	}
+
+	return owners, nil
+}
+
+func (r *ReplicaSet) GetInstance(fqn string) (*appsv1.ReplicaSet, error) {
+	o, err := r.getFactory().Get(r.gvr.String(), fqn, true, labels.Everything())
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
We don't need to have the full hydrated instance to retrieve controllers ie o.GetOwnerReferences()

Reply:
How would I only get the OwnerReferences? I haven't found existing code to only get specific attributes of a resource.

Reply:
```go
u, ok  := o.(*unstructured.Unstructured)
u.GetOwnerReferences()
```

Reply:
This was the golder tip for me to finally make the function _generic_. So I moved `GetOwners` to `generic.go`.

Reply:
@derailed, what do you think about the current implementation? I would be really happy if we could get this feature added to k9s.